python/tank/platform/engine.py

```diff
@@ -1896,7 +1901,7 @@ def _execute_post_context_change_hook(tk, previous_context, current_context):
     )
 
 
-def _start_engine(engine_name, tk, old_context, new_context):
+def _start_engine(engine_name, tk, new_context):
```

Comment:
`_start_engine` used to try to hide the complexity of context switching by muting None transitions. It won't anymore.